internal/models/api/common.go

```diff
@@ -0,0 +1,10 @@
+package api
+
+type ValidationError struct {
+    Fields []string
```

Comment:
Assuming we will always fail fast at the first validation error, lets keep this as `string` instead of `[]string`?

Reply:
Ok will make the change